### ISSUE TITLE
Add a 'contact' partial?

### ISSUE THREAD

**Issue by evanplaice** [fluentdesk/fluent-themes#39]
Not sure if dedicating a partial to `contact` info. Providing a default structure to display this info may improve DRY.

``` json
  "contact": {
    "email": "",
    "phone": "",
    "website": "",
    "other": [
      {
        "label": "",
        "flavor": "",
        "value": ""
      } 
    ]
  }
```

**Comment by hacksalot** [comment]
I've gone back and forth on this a bit but it's better to have a partial and not need it than need a partial and not have it, especially when building a new partial doesn't involve a lot of effort. This also mirrors the structure of the FRESH schema, where `contact` is a root-level element.

**Comment by evanplaice** [comment]
I agree. I suggested it because I think the basic template should just list all of the partials. Since you feel this way I'll add 2 more issues for sections that are currently missing but follow the same logic (ie location & social).

Then themes can customize the structure by adding their own. Such as the `skills` partial used in the 'modern' template.
